spec/models/comment_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
```

Comment:
- [ ] Kindly check on your comment model I got the error below when running the test:
![image](https://user-images.githubusercontent.com/61408860/190588721-fc4c9e69-496f-45aa-8f8d-ce45e211c33b.png)